packages/desktop-client/src/components/forms.tsx

```diff
@@ -23,25 +29,44 @@ export function SectionLabel({ title, style }) {
       {title}
     </View>
   );
+};
+
+interface FormLabelProps {
+  title: string;
+  id?: string;
+  htmlFor?: string;
+  style?: CSSProperties;
 }
 
-export function FormLabel({ style, title, id, htmlFor }) {
+export const FormLabel: React.FC<FormLabelProps> = ({
+  style,
+  title,
+  id,
+  htmlFor,
+}) => {
   return (
     <Text style={[{ fontSize: 13, marginBottom: 3, color: colors.n3 }, style]}>
       <label htmlFor={htmlFor} id={id}>
         {title}
       </label>
     </Text>
   );
+};
+
+interface FormFieldProps {
+  style?: CSSProperties;
+  children: React.ReactNode;
 }
 
-export function FormField({ style, children }) {
+export const FormField: React.FC<FormFieldProps> = ({ style, children }) => {
   return <View style={style}>{children}</View>;
-}
+};
 
 // Custom inputs
 
-export function Checkbox(props) {
+type CheckboxProps = React.HTMLProps<HTMLInputElement>;
```

Comment:
Potentially:
```suggestion
type CheckboxProps = Omit<React.HTMLProps<HTMLInputElement>, 'type'>;
```

(Also, should the `style` prop be updated?)